packages/react-hydrate/src/test/e2e.test.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
```

Comment:
Are we calling these `e2e` now?

Reply:
I dunno if it's a real convention, but it wasn't a test of any particular part so it felt appropriate here.